### PR TITLE
Assert with expect.arrayContaining and expect.objectContaining to check for correctness in tests.

### LINKED ISSUE #425: Flaky AccountHistory tests
#### What happened:

See https://github.com/DeFiCh/jellyfish/runs/2928506980#step:7:334

/assign @aikchun 
/triage accepted
/priority important-soon
/area jellyfish-api-core

### PR DESCRIPTION
**Pull request by aikchun**
#### What kind of PR is this?:

/kind fix

#### Which issue(s) does this PR fixes?:

Fixes #425

### CODE REVIEW

**Comment by aikchun** [comment]
@fuxingloh I am unable to replicate the test failing.

I did however use the `expect.arrayContaining` and `expect.objectContaining` strategy to make the assertions.

**Review by jingyi2811 — APPROVED** [review]

**Comment by jingyi2811** [comment]
> @fuxingloh I am unable to replicate the test failing.
> 
> I did however use the `expect.arrayContaining` and `expect.objectContaining` strategy to make the assertions.

I think

**Comment by jingyi2811** [comment]
Sorry, accidentally closed this PR. Reopened it back.

**Review by canonbrother — COMMENTED** [review]
overall good

File: packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts [hunk 1]
```diff
@@ -242,7 +242,16 @@ describe('listAccountHistory', () => {
 
     const history = await client.account.listAccountHistory()
 
-    expect(history.find((h) => h.type === 'UtxosToAccount' && h.amounts[0] === '4.97000000@DFI')).toBeTruthy()
+    expect(history).toEqual(
```
Comment (ivan-zynesis) [inline packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts:245]: This syntax definitely looks better.

I think it's not the root cause for the random test failure. The `history.find(absolutelyStrictPredicate)` should works fine, I guess could be the full history actually flooded by a lot of "blockReward". Occasionally, (for whatever reason) the test instance has lower processing power causing that row of history not included, outside the 100 rows limit. Add `no_rewards` option should fix it.

File: packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts [hunk 2]
```diff
@@ -242,7 +242,16 @@ describe('listAccountHistory', () => {
...
+      expect.arrayContaining([
```
Comment (ivan-zynesis) [inline packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts:246]: This syntax definitely looks better.

I think it's not the root cause for the random test failure. The history.find(absolutelyStrictPredicate) should works fine, I guess could be the full history actually flooded by a lot of "blockReward". Occasionally, (for whatever reason) the test instance has lower processing power causing that row of history not included, outside the 100 rows limit. Add no_rewards option should fix it.

**Review by ivan-zynesis — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts (hunk 2 above)
Comment (aikchun) [reply packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts:255]: Thanks. I've made this changes.

**Review by ivan-zynesis — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**